### DISCUSSION TITLE
Move note to another notebook

### DISCUSSION THREAD

**Discussion by hari-bo** [enchant97/note-mark#196] [Q&A] [answered]
Within the menu setting for notes i can add assets, change name, print, delete etc. But i can't find an option to move a note from notebook A to notebook B. 
Can someone give me a clue?

**Comment by enchant97** [comment, marked as answer]
You cannot currently move a note between books via a button. You can of course copy the contents between the old and new.

**Reply by hari-bo** [reply]
Will this be an feature in the future?

**Reply by enchant97** [reply]
I won't personally be implementing it. However, anyone is free to submit a PR to add it.

**Reply by hari-bo** [reply]
Thanks for the quick reply, too bad that's the only thing that's missing for a complete replacement.
